Makefile

```diff
@@ -1,2 +1,13 @@
 install:
-	go install ./cometmock
\ No newline at end of file
+	go install ./cometmock
+
+test-locally:
+	go test -timeout 600s ./e2e-tests -test.v 
+
+test-docker:
+	# Build the Docker image
+	docker build -f Dockerfile-test -t cometmock-test .
+
+	# Start a container and execute the test command inside
+	docker rm cometmock-test-instance || true
+	docker run --name cometmock-test-instance --workdir /CometMock cometmock-test go test -timeout 600s ./e2e-tests -test.v
```

Comment:
Not that familiar with docker but do we need something to tear down the docker after the tests run or is it done automatically?

Reply:
No, it dies after the command is finished executing, in this case `go test ...`